### DISCUSSION TITLE
Suggestion: Add a Reblog tag to Quick Tags/Quick Reblog

### DISCUSSION THREAD

**Discussion by kc446** [AprilSylph/XKit-Rewritten#384] [Suggestion Box]
### Concept

Quick Tags allows you to set tags for original posts and answered asks, but strangely not for reblogs, like the original Xkit. As a (pseudonymous) user who reblogs more than I actually post (a New Year's resolution I'm trying to set for myself), it would be nice to have an automatic tag set in Quick Reblog (doesn't have to be Quick Tags exclusively, of course) to keep my blog somewhat organized.
The original Xkit extension also had the ability to set automatic tags based on content (photo, audio, video, etc.), but that could be listed as a separate issue/request, or not at all, that's fine too. This is my first time offering a contribution/suggestion to another GitHub project so I wanted to keep it simple :) Thank you for reading!

**Comment by AprilSylph** [comment]
> The original Xkit extension also had the ability to set automatic tags based on content (photo, audio, video, etc.)

This is already present in Quick Reblog in the form of tag suggestions, which you can enable in Quick Reblog's options.

**Reply by kc446** [reply]
I meant having unique tags for media content already appended to the tag field by default, like how the original Xkit's One-Click Reblog feature adds the corresponding Auto Tagger tags in the field by default. Sorry if that wasn't made clear, and again, it's fine if that is pushed to a separate issue.

**Comment by AprilSylph** [comment]
Is a reblog tag really necessary when it's already possible to set a queue tag in Quick Reblog? I have reservations about adding more automatic tagging features than necessary, since Tumblr shows tags on reblog notifications by default now.

**Reply by kc446** [reply]
Having the option never hurts. I always assumed queues and reblogs were separate things since you can queue original posts too. That's why I personally don't have a queue tag, but I know some users may prefer to use a tag for queues or even tags for other media content, so I don't think it would be a bad thing to let the user have a little more customization with automatic tags. After all, Xkit was designed to give the user more control over their Tumblr experience :)

I can understand your reservations, though. I know I can get a bit lengthy with my tags sometimes since that's where i tend to leave my own silly little comments so they don't interfere or detract from the post itself. It doesn't really bother me on either side - honestly I would be _thrilled_ if someone reblogged one of my posts and left a few little thoughts of their own. It's the little things that make you smile :)

Of course, these are just my own thoughts. Everyone has their own way of tagging their posts.
